packages/next/client/head-manager.ts

```diff
@@ -93,7 +96,9 @@ export default function initHeadManager(initialHeadEntries: HeadEntry[]) {
 
   updateElements(
     elements,
-    initialHeadEntries.map(([type, props]) => createElement(type, props)),
+    initialHeadEntries
+      .filter(([type]) => type)
+      .map(([type, props]) => createElement(type, props)),
```

Comment:
This probably isn't right, but I simply filtered out any null `type` values here, which it seems to be null for react components (class/functional). Maybe there's a better way to handle this...

What this means is that this initial invocation of `updateElements` does NOT contain the react component that may have a bunch of meta/link elements in it, but then `updateHead` fires off and eventually adds them to the `<head>`.

<img width="1679" alt="Screen Shot 2020-10-09 at 10 14 39 PM" src="https://user-images.githubusercontent.com/6808172/95644934-f4da3e80-0a7f-11eb-8b08-1640bf35336d.png">
<img width="707" alt="Screen Shot 2020-10-09 at 10 21 38 PM" src="https://user-images.githubusercontent.com/6808172/95644936-f60b6b80-0a7f-11eb-86d5-adc72e32d117.png">